src/ontology/edits/oeo-social.omn

```diff
@@ -484,6 +491,48 @@ Class: OEO_00000431
         OEO_00000051
     
     
+Class: OEO_00030016
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An organizational energy producer is an organization that undertakes the generation of electricity and/or heat.\"",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "An organizational energy producer is an organization that undertakes the generation of electrical energy and/or heat.\"",
```
Since we have the class "electrical energy" and "electricity" is just its synonym...

Reply:
@l-emele just wrote in #381 it's not a synonym

Reply:
I'll merge, please create an issue if the wording here should be a significant problem

Reply:
I propose to reformulate:
_An organizational energy producer is an organization that undertakes the generation of  **energy**._

And is the backslash needed?